corehq/apps/app_manager/fixtures/mobile_ucr.py

```diff
@@ -112,6 +111,15 @@ def _get_apps(self, restore_state, restore_user):
 
         if app_aware_sync_app:
             apps = [app_aware_sync_app]
+        elif (
+            toggles.RESTORE_ACCESSIBLE_REPORTS_ONLY.enabled(restore_user.domain)
+            and restore_state.params.is_webapps
+            # only way to reliably know that this is a web apps restore, not live preview
+            and not restore_user.can_view_apps(restore_user.domain)
+        ):
+            for app in get_web_apps_available_to_user(restore_user.domain, restore_user._couch_user):
+                if not is_remote_app(app):
+                    apps.append(get_correct_app_class(app).wrap(app))
```

Comment:
Where is `apps` initialized? I see it in the `if app_aware_sync_app:` branch above, but if that executes this branch will not, I think.

Maybe a test or two would be good for this function?